cmd/server/main.go

```diff
@@ -26,6 +26,16 @@ func main() {
 			Value: "/var/lib/ocid/sandbox",
 			Usage: "ocid pod sandbox dir",
 		},
+		cli.Float64Flag{
```

Comment:
@mrunalp , this is for the case where the container config does not specify cpu and memory resources.
Should ocid just fail in that case or use some internal defaults?